README.md

```diff
@@ -2,18 +2,6 @@ Tipser Docs
 ---
 See the live docs at https://tipser.github.io/docs/
 
-run this to run locally:
```

Comment:
What's wrong with those section? Wasn't it helpful to know how to setup and run this docs project?